CMakeLists.txt

```diff
@@ -42,3 +47,4 @@ add_subdirectory(hart)
 add_subdirectory(instr)
 add_subdirectory(memory)
 add_subdirectory(simulator)
+add_subdirectory(riscv)
```

Comment:
Maybe change this module name to smth like app/sim_app etc?

Reply:
maybe later, not now